optaweb-vehicle-routing-frontend/src/store/client/reducers.ts

```diff
@@ -6,6 +6,7 @@ export const initialViewportState: UserViewport = {
   zoom: 2,
 };
 
+// eslint-disable-next-line @typescript-eslint/default-param-last
```

Comment:
I notice this eslint disable on several lines in the PR. Should the rule be disabled in .eslintrc?

Reply:
I think the rule makes sense in general. It's just incompatible with the reducer signature:

From https://redux.js.org/tutorials/fundamentals/part-3-state-actions-reducers:

> Reducers are functions that take the current state and an action as arguments, and return a new state result. In other words, `(state, action) => newState`.

and

> Reducers normally use ES6 default argument syntax to provide initial state: `(state = initialState, action)`.